### DISCUSSION TITLE
Navigation bar using ReactPy and Bootstrap 5.2

### DISCUSSION THREAD

**Discussion by fsmosca** [idom-team/idom#1124] [Share]
A navbar demo, the [source code](https://github.com/fsmosca/ReactPy-Navbar) is in my github repository.

## Big screen

![image](https://github.com/reactive-python/reactpy/assets/22366935/3a47e01b-c1df-46e9-b0f8-1bad79a173d6)

## Small screen.

![image](https://github.com/reactive-python/reactpy/assets/22366935/650932c7-84a8-419b-804a-844775bae661)

**Comment by fsmosca** [comment]
## Development notes

I use fastapi backend.

navbar.py

```python
from reactpy import component, html
from reactpy.backend.fastapi import configure, Options
from reactpy_router import route, simple
from fastapi import FastAPI

HEADER_TITLE = 'ReactPy-Navbar'
...

app = FastAPI()
configure(app, root, options=Options(head=html.head(html.title(HEADER_TITLE))))
```

and normally just use the following requirements.txt content.

```
reactpy
reactpy-router
```

and run the app with

```
uvicorn navbar:app --reload
```

That requirements.txt has an issue though, it cannot see `reactpy_router`.

The solution is to use

```
reactpy[fastpapi]
```

in the requirements.txt

I am on windows 10 OS,

**Reply by rmorshea** [reply]
It would be really cool to have some of this released as a `reactpy-bootstrap` package!

**Reply by fsmosca** [reply]
Not this time. But I will.
